src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registerUserWithInsufficientAge_ThrowsRegistrationException() {
```

Comment:
Do we test  `registerUserWithInsufficientAge` from RegistrationServiceImpl class? No. We do not have method with name `registerUserWithInsufficientAge` in RegistrationServiceImpl.

![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/99d78f48-2456-4e1c-9ecc-a73689620c97)